server/chat-plugins/chat-monitor.ts

```diff
@@ -642,13 +642,74 @@ export const commands: ChatCommands = {
 			this.parse(`/join view-filters`);
 		},
 		help(target, room, user) {
-			this.parse(`/help filter`);
+			this.run(`filterhelp`);
+		},
+		test(target, room, user, connection, cmd) {
+			this.privatelyCheckCan('lock');
+			if (room && ['staff', 'upperstaff'].includes(room.roomid)) this.runBroadcast(true);
+			let [message, monitorName] = target.split('|').map(x => x.trim());
+			if (!monitorName && cmd !== 'test') {
+				monitorName = cmd.slice(4);
+				if (!Chat.monitors[monitorName] && Chat.monitors[monitorName + 'filter']) {
+					monitorName = monitorName + 'filter';
+				}
+			}
+			// namefilter doesn't have a monitor function
+			if (!(monitorName && message) || !Chat.monitors[monitorName]?.monitor) {
+				return this.run((Chat.commands.filter as any).testhelp);
+			}
+			const monitor = Chat.monitors[monitorName].monitor!;
+			const lcMessage = Chat.stripFormatting(message
+				.replace(/\u039d/g, 'N')
+				.toLowerCase()
+				// eslint-disable-next-line no-misleading-character-class
+				.replace(/[\u200b\u007F\u00AD\uDB40\uDC00\uDC21]/g, '')
+				.replace(/\u03bf/g, 'o')
+				.replace(/\u043e/g, 'o')
+				.replace(/\u0430/g, 'a')
+				.replace(/\u0435/g, 'e')
+				.replace(/\u039d/g, 'e'));
```

Comment:
I take it you're not planning to break this out into its own function with this PR? I can do that myself then.

Reply:
Yeah it's not really relevant to this PR so I didn't feel too comfortable compacting it